SpatialGDK/Source/SpatialGDK/Private/EngineClasses/AbstractLBStrategy.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
nit: these files shouldn't be in `EngineClasses`. That folder is supposed to be for UnrealGDK implementations of specific classes (ie. USpatialNetConnection), not generic UObjects.

Reply:
Moved to /LoadBalancing/ for now.